README.md

```diff
@@ -1,2 +1,79 @@
+[![Build Status](https://badge.buildkite.com/9e0e6c88972a3248a0908506d6946624da84e4e18c0870c4d0.svg)](https://buildkite.com/rust-vmm/vm-virtio-ci)
+
 # vm-virtio
-virtio implementation
+
+The vm-virtio crate provides a [virtio](https://www.oasis-open.org/committees/tc_home.php?wg_abbrev=virtio)
+device implementation:
+
+* A virtio virtqueue and Descriptor chain API
+* A virtio device trait
+
```

Comment:
Please also add some details about the trait and specify how this is going to be used.

Reply:
I expanded the README, let me know if that looks better.